### PR TITLE
README.md: Document huffman python bindings

### PR DESCRIPTION
**Pull request by ChillerDragon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Documentation
   the low-level file format of Teeworlds/DDNet demos (replays).
 - [huffman](doc/huffman.md). Homebrew compression format using [Huffman
   coding](https://en.wikipedia.org/wiki/Huffman_coding), used in demos and over
-  the network.
+  the network. Comes as a rust create but also has [python bindings](./uniffi/huffman/)
```
Comment (heinrich5991) [inline README.md:32]: This is the documentation section, so the link doesn't make sense here. There's an extra `./` in front.

There's already a link to the uniffi stuff below, so I don't think we need to add one.

File: uniffi/huffman/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# libtw2_huffman
+
+Python bindings for huffman.
```
Comment (heinrich5991) [inline uniffi/huffman/README.md:3]: What is "huffman"? This summary doesn't really tell me anything that I can't tell from the directory listing.

File: uniffi/huffman/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Building from source
+
+```bash
+python -m pip install --upgrade build
+python -m build
+pip install dist/libtw2_huffman-*.tar.gz
+```
+
+## Install from pypi
```
Comment (heinrich5991) [inline uniffi/huffman/README.md:13]:
```suggestion
## Install from PyPI
```

File: uniffi/huffman/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# libtw2_huffman
+
+Python bindings for huffman.
+
+## Building from source
+
+```bash
+python -m pip install --upgrade build
+python -m build
+pip install dist/libtw2_huffman-*.tar.gz
```
Comment (heinrich5991) [inline uniffi/huffman/README.md:10]:
```suggestion
python -m pip install dist/libtw2_huffman-*.tar.gz
```

**Review by heinrich5991 — COMMENTED** [review]
A README.md is nice. Currently, the summary doesn't tell me anything, but the installation instructions and sample usage are useful.

File: README.md (hunk 1 above)
Comment (ChillerDragon) [reply README.md:32]: Well I did not find it the first time I was looking for it. I searched for huffman and python in the readme and it did not help. I don't think it is obvious that the python huffman bindings are located in the uniffi directory.

File: uniffi/huffman/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# libtw2_huffman
+
+Python bindings for huffman (lossless data compression).
```
Comment (heinrich5991) [inline uniffi/huffman/README.md:3]:
```suggestion
Python bindings for `libtw2-huffman` which implements a homebrew compression
format using huffman coding that is used in Teeworlds/DDNet networking and demo
file format.
```

The summary is still not meaningful IMO. It's important to give context.

**Review by heinrich5991 — APPROVED** [review]
Thanks!

**Pull request merged**